packages/example_hardhat/README.md

```diff
@@ -0,0 +1,79 @@
+# <PACKAGE>
+
+*Include a high level description of this package here*
+
+## Running the project locally
+
+Install dependencies with `yarn`
+
+Testing
+Use `yarn test` inside `packages/<package>` to run tests locally for just this package
+
+Coverage
+Run `yarn coverage` 
```

Comment:
Can we use the actual workspace feature (from root) which is suppose to help working with packages ?
```
yarn workspace @eniblock/mypackage test
```

Reply:
Yep I'll update it with instructions (i) from root, (ii) from inside package